### ISSUE TITLE
Make it more obvious what Northstar is for someone completely new to it

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/NorthstarTF#10]
Someone who may have never heard about Northstar (and maybe isn't all that familiar with Titanfall2), coming to the webpage and seeing 

> Titanfall|2 modding and custom server framework

![image](https://user-images.githubusercontent.com/40122905/203309742-d13bc30c-31e4-482b-bed8-37c1ba0a4b0a.png)

may not mean anything to them. The website should somewhat specify what Northstar is and/or what it allows you to do.

For example having some sub heading or paragraph on the main page that says something like

> Custom servers, custom skins, custom gamemodes. Supercharge your Titanfall|2 experience!

would give a lot more info already. (Just an example feel free to propose other ideas.)

Similarly having [Smurfson's initial Northstar trailer](https://www.youtube.com/watch?v=en06Y6CPMQg) on the main page would help convey more info about what Northstar is to someone completely new to it.

**TL;DR:** Current webpage gives to little info about what Northstar is to someone completely new to it (and Titanfall2 as a whole).
